src/App.css

```diff
@@ -9,103 +9,105 @@
   background-color: #221f20;
   min-height: 344px;
 }
-.search-header-login {
+.navigation {
   background-color: #221f20;
   font-size: 40px;
-  min-height: 70px;
 }
 
-
 .logo-itviec {
   padding-top: 10px;
+  padding-bottom: 10px;
   height: 50px;
 }
 
 .search-form-wrapper {
   margin: 50px auto 50px;
 }
 .search-section-wrapper {
-  width: 100%;
-
-  height: 60px;
-  line-height: 60px;
   background: white;
   border: 1px solid #bebebe;
+  height: 60px;
+  line-height: 60px;
+  width: 100%;
 }
 .search-field-wrapper {
-  margin-left: 8px;
+  align-items: center;
+  display: flex;
   height: 100%;
+  margin-left: 8px;
 }
 
 .search-box {
+  border: 0;
+  font-size: 18px;
   height: 90%;
-  width: 100%;
   line-height: 30px;
-  font-size: 18px;
   margin-top: 1px;
-  border: 0;
+  width: 100%;
 }
 .search-box:focus {
   border: 0;
 }
 .search-button {
-  width: 100%;
-  height: 61px;
-  font-size: 20px;
-  border-radius: 0;
   background-color: #ea1e30;
-
-  color: white;
   border: 0;
+  border-radius: 0;
+  color: white;
+  font-size: 20px;
+  height: 61px;
+  width: 100%;
 }
 .search-button:hover {
   background-color: #d82727;
 }
 
 .job-list {
-  padding: 0 10px;
   background-color: white;
   border: 1px solid #d8d8d8;
-
   margin-top: -77px;
+  padding: 0 10px;
   padding-bottom: 10px;
 }
 
 .middle {
-  height: 100%;
   display: flex;
   flex-direction: column;
+  height: 100%;
   justify-content: center;
-  align-items: center;
 }
-.login-form {
-  padding: 50px;
+.white-container {
   background-color: white;
   border: 1px solid #d8d8d8;
-  max-width: 1024px;
-  min-width: 700px;
+  overflow: auto;
+  padding: 50px;
+}
+.login-title-box {
+  align-items: center;
+  background: black;
+  display: flex;
+  justify-content: center;
+  margin-bottom: 20px;
+  padding-top: 10px;
+  padding-bottom: 10px;
 }
-h1 {
+.login-title {
+  color: white;
   font-size: 30px;
-  color: #221f20;
-  line-height: 35px;
-  padding-top: 20px;
-  padding-left: 10px;
-  padding-bottom: 20px;
-  height: auto;
+  margin-left: 20px;
+  text-align: center;
 }
 .job-content {
-  transition: all 0.25s ease;
   border-top: 2px solid rgba(255, 192, 121, 0.5);
   border-bottom: 2px solid rgba(255, 192, 121, 0.5);
   background: #fff1e1 !important;
   padding: 27px 0;
+  transition: all 0.25s ease;
 }
 .job-content:hover {
   background: white !important;
-  z-index: 5;
   border-color: transparent;
   box-shadow: 0 0 5px 1px #bcbcbc;
+  z-index: 5;
```

Comment:
why 5?

Reply:
yeah that's unnecessary i removed it too